dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like you missed an "All rights reserved"?

Reply:
Actually, seeing at least three instances, might be worth a re-scrub.

Reply:
I had checked for two spaces before "all rights reserved".  Fixed.